src/services/CertificateGenerationService.ts

```diff
@@ -563,6 +566,59 @@ class CertificateGenerationService {
 
         return defectString;
     }
+    /**
+     * To fetch trailer registration
+     * @param vin The vin of the trailer
+     * @param make The make of the trailer
+     * @returns A payload containing the TRN of the trailer and a boolean.
+     */
+     public async getTrailerRegistrationObject(vin: string, make: string) {
+        const config: IInvokeConfig = this.config.getInvokeConfig();
+        const invokeParams: any = {
+            FunctionName: config.functions.trailerRegistration.name,
+            InvocationType: "RequestResponse",
+            LogType: "Tail",
+            Payload: JSON.stringify({
+                httpMethod: "GET",
+                path: `v1/trailers/${vin}?make=${make}`,
+                pathParameters: {
+                    proxy: `v1/trailers`
+                },
+                queryStringParameters: {
+                    make
+                }
+            }),
+        };
+        const response = await this.lambdaClient.invoke(invokeParams);
+        try {
+             const payload: any = this.lambdaClient.validateInvocationResponse(response);
+             const trailerRegistration = JSON.parse(payload.body) as ITrailerRegistration;
+             return {Trn: trailerRegistration.trn, isTrailer: true};
+         } catch (err) {
+             if (err.statusCode === 404) {
+                console.warn(`vinOrChassisWithMake not found ${vin + make}`);
```

Comment:
I don't want to be a pain, but this isn't a warning.  Console.debug would be right.

Reply:
done.